.golangci.yml

```diff
@@ -52,6 +53,14 @@ issues:
       text: "redefines-builtin-id"
 
 linters-settings:
+  importas:
+    # Do not allow unaliased imports of aliased packages.
+    no-unaliased: true
+
+    alias:
+      - pkg: github.com/opencontainers/image-spec/specs-go/v1
+        alias: ocispec
```

Comment:
I'd posit that `imagespec` would be a better choice here since we also import the `runtimespec` package (and that's also an OCI spec).

Reply:
But other than bikeshedding the alias, I'm :+1: on keeping it consistent throughout the codebase.

Reply:
Yeah, I was slightly on the fence, but looked across repositories, and most seemed to use `ocispec`.

Looking at how it's used, `ocispec` is not too ambiguous/confusing;

- `ocispec.Image`
- `ocispec.ImageConfig`
- `ocispec.Platform`